SupportBank/Program.cs

```diff
@@ -1,46 +1,47 @@
 ﻿using System;
-using System.Collections.Generic;
-using System.Globalization;
-using System.IO;
-using CsvHelper;
+using System.Linq;
 
 namespace SupportBank
 {
     class Program
     {
-        private static Dictionary<string,Account> _accounts;
-        
         static void Main(string[] args)
         {
-            _accounts = new Dictionary<string, Account>();
-            CSVReader();
-            foreach (var account in _accounts.Values)
+            var bs = new Base();
```

Comment:
It's best practice to avoid abbreviations in variable names as it makes the code less readable 🙂